utils/test/garden-test-utils.tsx

```diff
@@ -34,6 +34,29 @@ const customRtlRender = (ui: React.ReactElement, options?: any) =>
 const customDarkRender = (ui: React.ReactElement, options?: any) =>
   render(ui, { wrapper: DarkThemeWrapper, ...options });
 
+const MODE_TO_RENDER_FN_MAP = {
+  dark: customDarkRender,
+  light: customRender
+};
+/**
+ * A utility fn that returns the correct render function
+ * for the mode. Useful when using `it.each()`.
+ *
+ * @throws Throws an error if mode is not recognized.
+ *
+ * @param mode {string} The color mode.
+ * @returns {Function} Render function associated with the mode.
+ */
+export function getRenderFn(mode: string) {
```

Comment:
Should `renderRtl` also be incorporated in this convenience function? Perhaps a `getRenderFn({ mode, rtl }: { string, boolean })` type signature 🤷

Reply:
Good idea! 🔥 

[chore: add isRTL support](https://github.com/zendeskgarden/react-components/pull/1819/commits/bfeeed60f7ab433a8c2a58958a1e9303af4cdf55)